### PR TITLE
Add subscribe/sign-in links to notification messages

### PR DESCRIPTION
**Pull request by keirog**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -50,8 +50,8 @@ const idProperties = {
 };
 
 const nNotificationMsgs = {
-	followAnon: 'Please subscribe or sign in to follow this topic.',
-	saveAnon: 'Please subscribe or sign in to save this article.',
+	followAnon: 'Please <a href="/products?segID=400863&segmentID=190b4443-dc03-bd53-e79b-b4b6fbd04e64">subscribe</a> or <a href="/login">sign in</a> to follow this topic.',
```
Comment (laurieboyes) [inline main.js]: Is it worth us storing this url in res.locals so we don't have to duplicate it everywhere? We use it in product tour redirect as well

Comment (keirog) [inline main.js]: Yeah good call. Was just rushing

Comment (laurieboyes) [inline main.js]: Happy to :+1: if we're aiming for a quickfix

**Comment by laurieboyes** [comment]
bon :+1:

**Pull request merged**